packages/hooks/src/useResponsive/index.ts

```diff
@@ -58,16 +58,19 @@ export function configResponsive(config: ResponsiveConfig) {
 export function useResponsive() {
```

Comment:
这个判断可以前置到这里么

```suggestion
export function useResponsive() {
  if (typeof window === "undefined") return
```

不过这样会被 lint 识别成违反了 Hooks 的使用规则

Reply:
I don't think that is allowed because of other hook called just after:

https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

Reply:
Yes. So either we add a eslint ignore comment or we need to add checks here and there, which may increase the maintaining difficulty in the future. But I think you are right. This hook is simple enough so maybe we don't need to worry too much about maintainability.